ttnn/cpp/ttnn/operations/eltwise/unary/unary_composite.hpp

```diff
@@ -86,6 +87,36 @@ struct Power{
         return detail::power_fp(DefaultQueueId, input_tensor, exponent, memory_config, optional_output_tensor);
     }
 };
+template <UnaryCompositeOpType unary_comp_op_type>
+struct Unary_composite_ops
+{
+    static ttnn::Tensor execute_on_worker_thread(
+        const Tensor& input_tensor,
+        const std::optional<MemoryConfig>& memory_config = std::nullopt)
+        {
+            auto op_type = UnaryCompositeFunction::get_function_type1(unary_comp_op_type);
+            return op_type(input_tensor, memory_config);
+        }
+
+    static ttnn::Tensor execute_on_worker_thread(
+        const Tensor& input_tensor,
+        float value_1,
+        float value_2,
+        const std::optional<MemoryConfig>& memory_config = std::nullopt)
+        {
+            auto op_type = UnaryCompositeFunction::get_function_type2(unary_comp_op_type);
+            return op_type(input_tensor, value_1, value_2, memory_config);
+        }
+
+    static ttnn::Tensor execute_on_worker_thread(
+        const Tensor& input_tensor,
+        int value_1,
+        const std::optional<MemoryConfig>& memory_config = std::nullopt)
+        {
+            auto op_type = UnaryCompositeFunction::get_function_type3(unary_comp_op_type);
+            return op_type(input_tensor, value_1, memory_config);
+        }
+};
```

Comment:
@eyonland , adding queue_id / optional_output_tensor separately, correct?

Reply:
Yea this is part of M2 as defined in [the status ](https://docs.google.com/document/d/1B74vJPjpUCjM3c3K3QOShQznDUNCdBAlcu5J3V2671Y/edit)